syntax = "proto3";
option optimize_for = CODE_SIZE;

package checkpoint;

service CheckpointService {
	rpc Create(CreateCheckpointRequest) returns (CreateCheckpointResponse);
}

message CreateCheckpointRequest {
    string container =1;
    string checkpoint =2;
    string dir =3;
}

message CreateCheckpointResponse {
	uint32 cc = 1;
	string errmsg = 2;
}
